adops/line-item-creation.md

```diff
@@ -1,61 +1,73 @@
+- What Prebid scenarios do you need now and in the forseeable future? Is Prebid.js all you need? Or do you have an [AMP](/formats/formats.html#amp) version of your site? Do you have a mobile app?
+- How many line items can you afford to create? Very high [price granularities](/adops/price-granularity.html) can consume a large number of line items, and some ad servers have strict limits on the number of active and/or historic line items.
+
+Here's a set of basic recommendations to use as a starting point:
+
+1. **Do you run Prebid.js only?** Create 2 sets of line items: one set for instream-video and another set for everything else.
+1. **Do you run Prebid.js and AMP?**
+    1. For now, you'll need to have two sets of line items: one for instream video, and one that uses the PUC for everything else.
+    1. In the future, there will be an option to run three sets of line items to utilize the [dynamic creative](/adops/creative-considerations.html#prebidjs-dynamic-creatives) for Prebid.js.
+1. **Do you run Prebid.js and Prebid Mobile?**
+    1. If you're using multiformat banner and "in-renderer" video (formerly "outstream"), you will need to have a separate set of adunits and line items for mobile. This is because Prebid Mobile does not support javascript renderers like Prebid.js, instead replying the ad server SDK, which requires them to be treated the same as instream video. So in this scenario you'll have 4 sets of line items: Prebid.js instream, Prebid.js banner+native+noninstream, mobile video, and mobile banner. If you run mobile native or rewarded video, each will be another set of line items.
+    1. If you have chosen the Prebid-Rendered approach for your mobile apps in order to get SKAdNetork and advanced adunit support, you'll want to have separate sets of line items for mobile and Prebid.js as described for the multiformat item just above.
+    1. If you're running Rewarded Video, you'll want to define which adunits run that format and have a separate set of video line items that target those units. This is because rewarded video has a special creative type.
+1. **Or just Prebid Mobile?**
+    1. As a base, you'll want one set of line items that covers all video (instream and non-instream), and another set for banner.
+    1. If you run native, you'll want another set of line items for them.
+    1. And then if you run rewarded video, another set of line items for those ad units.
 
-### At a Glance
+Yes, setting up line items for Prebid Mobile is complicated. We recommend reading the "Ad Operations Guidance" section of the
+specific mobile integration method you're using:
 
-These tables show the Prebid line item recommendations and requirements. The following sections provide more details on each.
```

Comment:
The formatting of this list looks incorrect in the preview. It looks like the code snippet. 

There are also incomplete placeholders for the sections' URLs

Reply:
Fixed - thanks!